assignment2.txt

```diff
@@ -0,0 +1,10 @@
+Jack Rosen
+
+1. They  cannot be combined into one statement because they will interfere with each other.
```

Comment:
Mmm, not exactly - they can be combined into a single line of code as long as each operator has the required amount of operands.